lib/src/cli/flutter_cli.dart

```diff
@@ -165,14 +167,15 @@ class Flutter {
         void noop(String? _) {}
         final target = DirectoryGeneratorTarget(Directory(p.normalize(cwd)));
         final workingDirectory = target.dir.absolute.path;
+        final relativePath = p.relative(workingDirectory, from: initialCwd);
 
         stdout?.call(
-          'Running "flutter test" in ${p.dirname(workingDirectory)}...\n',
+          'Running "flutter test" in .${p.context.separator}$relativePath...\n',
```

Comment:
**Nit:**

We should include a space between the `$relativePath` and the `...` that indicate progress. To avoid confusion between the directory and the `...`.

```suggestion
          'Running "flutter test" in .${p.context.separator}$relativePath ...\n',
```

Reply:
I like it! It would make it easier to read